src/org/rascalmpl/library/lang/rascal/tests/library/lang/json/JSONIOTests.rsc

```diff
@@ -69,4 +70,29 @@ test bool originTracking() {
    }
 
    return true;
+}
+
+data Cons = cons(str bla = "null");
+
+test bool dealWithNull() {
```

Comment:
seeing the changes in JsonIOReader, I expected more tests.

Reply:
added tests for all new features: null, Maybe, accurate parse errors, parsing/unparsing string fields